src/compiler/core.ts

```diff
@@ -2882,3 +2882,57 @@ export function isNodeLikeSystem(): boolean {
         && !(process as any).browser
         && typeof module === "object";
 }
+
+
+/** @internal */
+export interface StackSet<T> {

```

Comment:
Nit: maybe just put this in `utilities` unless we ever need it elsewhere.

Reply:
Sure, I honestly don't know what the difference is between any of these files anymore.